kvwrapper_etcd/kvwrapper_etcd.go

```diff
@@ -52,7 +67,11 @@ func (e EtcdWrapper) GetVal(key string) (*kvwrapper.KeyValue, error) {
 
 // GetVal returns a []KeyValue found at key
 func (e EtcdWrapper) GetList(key string, sort bool) ([]*kvwrapper.KeyValue, error) {
-	r, err := e.client.Get(key, sort, true)
+	options := &etcd.GetOptions{
+		Sort:      sort,
+		Recursive: true,
+	}
+	r, err := e.kapi.Get(context.Background(), key, options)
 	if err != nil {
 		if strings.HasPrefix(err.Error(), "501:") {
```

Comment:
We just need to distinguish between etcd being down and `key not found` which we won't treat as an actual error in our client code